tools/autograd/derivatives.yaml

```diff
@@ -220,20 +226,24 @@
   self: maybe_multiply(grad, beta.conj())
   mat: grad.ger(vec.conj()) * alpha.conj()
   vec: mat.t().conj().mv(grad) * alpha.conj()
+  result: maybe_multiply(self_t, beta) + maybe_multiply(mat_t.mv(vec_p), alpha) + maybe_multiply(mat_p.mv(vec_t), alpha)
 
 - name: addr(Tensor self, Tensor vec1, Tensor vec2, *, Scalar beta=1, Scalar alpha=1) -> Tensor
   self: maybe_multiply(grad, beta.conj())
   vec1: grad.mv(vec2.conj()) * alpha.conj()
   vec2: grad.t().mv(vec1.conj()) * alpha.conj()
+  result: maybe_multiply(self_t, beta) + maybe_multiply(vec1_t.outer(vec2_p), alpha) + maybe_multiply(vec1_p.outer(vec2_t), alpha)
 
 - name: affine_grid_generator(Tensor theta, int[] size, bool align_corners) -> Tensor
   theta: affine_grid_generator_backward(grad, size, align_corners)
 
 - name: alias(Tensor(a) self) -> Tensor(a)
   self: grad
+  result: auto_element_wise
```

Comment:
Is `self_t` more efficient? Otherwise the codegen will name this do `self_t.conj().conj()`, right?